common/services/person.js

```diff
@@ -1,15 +1,14 @@
 const { mapKeys, mapValues, omitBy, isNil } = require('lodash')
 
-const apiClient = require('../lib/api-client')()
-
+const BaseService = require('./base')
 const unformat = require('./person/person.unformat')
 
 const relationshipKeys = ['gender', 'ethnicity']
 const dateKeys = ['date_of_birth']
 const noPersonIdMessage = 'No person ID supplied'
 
-const personService = {
-  format(data) {
+class PersonService extends BaseService {
+  static format(data) {
```

Comment:
Don't think we need the static here - is this some Typescript linting that you have on by chance?